### PR TITLE
feat[added search result page]

### PR DESCRIPTION
**Pull request by Radheshyam1234**
Fronttube search feature

### CODE REVIEW

File: src/Components/SearchResults/SearchResultPage.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        searchedItem.toLowerCase().includes(category.toLowerCase()) ||
+        searchedItem.toLowerCase().includes(description.toLowerCase()) ||
+        searchedItem.toLowerCase().includes(title.toLowerCase())
+      );
+    });
+  };
+
+  useEffect(() => {
+    console.log(searchedItem);
+    console.log(FilteredData());
```
Comment (simran61) [inline src/Components/SearchResults/SearchResultPage.js]: You can remove console.log if not in use.

**Review by simran61 — APPROVED** [review]
Looks good. Clean code.

**Review by anamkaa — APPROVED** [review]
Good variable naming and cood looks clean , can be merged.

**Review by irajbhinde — COMMENTED** [review]
image

After clicking on the snackbars, the search bar css seems to be breaking, kindly look into it

**Pull request merged**